hubot-rocketchat/docker-compose.yml

```diff
@@ -55,20 +60,24 @@ services:
     - VIVA_REMINDER_SCHEDULER=${VIVA_REMINDER_SCHEDULER}
     - VIVA_REPORT_SCHEDULER=${VIVA_REPORT_SCHEDULER}
     volumes:
-    - ./packages:/home/hubot/packages
-    - ./scripts:/home/hubot/scripts
+    - ./packages:/hubot/packages
+    - ./scripts:/hubot/scripts
+    volumes_from:
+      - socket
   redis:
     image: cusdeb/redis:5.0-amd64
-    networks:
-    - rocketchat_rocketchat-network
     environment:
     # Enable AOF
     - REDIS_CONF_appendonly=yes
     # Disable RDB
     - REDIS_CONF_save=
     - REDIS_CONF_loglevel=notice
+    - REDIS_CONF_unixsocket=/tmp/socket/redis.sock
```

Comment:
* Let's create the sock file in `/var/run`
* Let's give it less generic name, for example, `hubot-redis.sock`